fms_parser.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/PuerkitoBio/goquery"
```

Comment:
I'd also put the third-party libraries at the end - for example, https://github.com/lethosor/TBA-uploader/blob/2407c80e7394110b443f5a5776184daec87e5072/fms_download.go#L14. Not sure how official this is - goimports does this, but gofmt doesn't.